### PR TITLE
Q7 V1

### PR DESCRIPTION
**Pull request by NaveedGujral**

### CODE REVIEW

File: models/models.js [hunk 1]
```diff
@@ -50,6 +49,21 @@ exports.selectAllCommentsByArtId = (artId) => {
...
+    const { username, body } = reqBody
+
+    let sqlStr = `INSERT INTO comments ( author, body, article_id ) VALUES ( $1, $2, $3 ) RETURNING *`
+
+    const valArray = [ username, body, artId ]
+
+    return db.query(sqlStr, valArray)
+    .then((table) => {
+        const comment = table.rows[0]
+        // console.log(comment)
```
Comment (MKD01) [inline models/models.js]: Dont forget to remove console.logs

File: __tests__/integration-tests.test.js [hunk 2]
```diff
@@ -138,6 +138,57 @@ describe('GET /api/articles/:article_id tests', () => {
...
+        return request(app)
+        .post("/api/articles/not-an-id/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request")
+        })
+    });
+});
+
```
Comment (MKD01) [inline __tests__/integration-tests.test.js:239]: Really good testing so far, but there are a few more tests that should be added for this endpoint.

What should happen when a user makes a request with an incomplete/incorrect request body?
What should happen when a user makes a request with a username that does not exist?

**Review by MKD01 — CHANGES_REQUESTED** [review]
Really good work, but I would like to see the missing tests added before you merge this in, so please re-submit this pr once the tests have been added.

File: __tests__/integration-tests.test.js [hunk 3]
```diff
@@ -138,6 +138,98 @@ describe('GET /api/articles/:article_id tests', () => {
...
+          } 
+        return request(app)
+        .post("/api/articles/not-an-id/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request")
+        })
+    });
+});
```
Comment (aloush-dev) [inline __tests__/integration-tests.test.js]: I think it should still be a 200 if the correct properties are there and just ignore the incorrect ones.
It should only be a 400 is username or body is missing

**Review by aloush-dev — APPROVED** [review]
Awesome job.
Please see the comment I left about one of the tests but merge when ready.

**Review by aloush-dev — APPROVED** [review]
Awesome job.
Please see the comment I left about one of the tests but merge when ready.

**Pull request merged**